packages/native-machine-id/package.json

```diff
@@ -9,11 +9,13 @@
     "bootstrap": "npm run compile",
     "pretest": "npm run compile",
     "install": "node-gyp rebuild",
+    "typecheck": "tsc --noEmit",
     "test": "mocha",
     "test-cov": "nyc -x \"**/*.spec.*\" --reporter=lcov --reporter=text --reporter=html npm run test",
     "test-ci": "npm run test-cov",
     "lint": "eslint . && prettier --check .",
```

Comment:
We'll also want to add prettier as a dev dependency and add a reformat script, I think